webapp/test/test_updates.py

```diff
@@ -62,49 +62,62 @@ def setup_api(self, load_cache):
 
     def test_process_repos(self):
         """Test repos processing."""
-        response = {}
-        repo_ids = self.updates_api._process_repositories(UPDATES_JSON, response)
-        assert len(repo_ids) == 1
+        repositories_list, repo_ids = self.updates_api._get_repository_list(UPDATES_JSON)
+        releasever, repo_ids = self.updates_api._get_releasever(UPDATES_JSON, repo_ids)
+        basearch, repo_ids = self.updates_api._get_basearch(UPDATES_JSON, repo_ids)
+        assert repositories_list == ['rhel-7-server-rpms']
+        assert releasever == '7Server'
+        assert basearch == 'x86_64'
+        assert repo_ids == {1}
```

Comment:
it no more tests self.updates_api._process_repositories() but 3 different methods

Reply:
Well, these methods depend on each other and relate to repository_list processing so I kept them together. Should I split it to separate tests or just rename current test?

Reply:
Ok, I've created a separate tests for the new functions. It will be better.

Reply:
that's what I mean; keep test for old method and create one for new